server/migrations/20200806170620-migration-skeleton.js

```diff
@@ -0,0 +1,164 @@
+      first_name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      last_name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      date_of_birth: {
+        type: Sequelize.DATE,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+          isDate: true,
+        },
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+          isEmail: true,
+        },
+      },
+      is_underage: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+        defaultValue: false,
+      },
+      street: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      city: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      country: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      allergies: {
+        type: Sequelize.TEXT,
+      },
+      accepts_tos: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      registration_status: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+        defaultValue: 'New',
+      },
+      displayed: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+        defaultValue: true,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    });
+    await queryInterface.createTable('payments', {
+      amount_paid: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      payment_date: {
+        type: Sequelize.DATE,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+          isDate: true,
+        },
+      },
+      type_of_payment: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        validate: {
+          notEmpty: true,
+        },
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    });
+  },
+
+  down: async (queryInterface, Sequelize) => {
+    await queryInterface.dropTable('payments');
+    await queryInterface.dropTable('attendants');
+    await queryInterface.dropTable('instruments');
+  },
+};
```

Comment:
Is there any way to reference the model files to avoid having the schema definition in two places? It's probably not important for now really, but I'd be curious to know whether it's possible